Comp2023-Imported/src/main/java/frc/robot/subsystems/Swerve.java

```diff
@@ -505,9 +514,12 @@ public Pose2d getPose( )
 
   public void resetOdometry(Pose2d pose)
   {
-    DataLogManager.log("heading " + m_heading);
-    DataLogManager.log("positions " + getPositions( ));
-    DataLogManager.log("pose " + pose);
+    DataLogManager.log("heading!!!!!!!!!!!!: " + m_heading);
+    for (int i = 0; i < getPositions( ).length; i++)
+    {
+      DataLogManager.log("positions!!!!: " + i + " " + getPositions( )[i]);
```

Comment:
The pose is useful, but I'm not sure logging the individual swerve module positions is helpful. We should discuss.